jinja2/loaders.py

```diff
@@ -160,7 +160,7 @@ class FileSystemLoader(BaseLoader):
     def __init__(self, searchpath, encoding='utf-8', followlinks=False):
         if isinstance(searchpath, string_types):
             searchpath = [searchpath]
-        self.searchpath = list(searchpath)
+        self.searchpath = [str(x) for x in list(searchpath)]
```

Comment:
Seems like this will still fail if searchpath is a single value. It's not `string_types`, so instead you'll try to iterate over a path.